docs/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,133 @@
+<!--
+Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+
+SPDX-License-Identifier: curl
+-->
+
+# Infrastructure in the curl project
+
+Overview of infrastructure we maintain, host and run in the project for the
+project.
+
+## git repository
+
+Since 2010, the main curl git repository has been hosted by GitHub, available
+at https://github.com/curl/curl.
+
+We also use the issue tracker, pull requests and discussions on GitHub.
+
+curl has an "enterprise account" on GitHub and is an "organization" on the
+site.
+
+We accept sponsorship via GitHub Sponsors.
+
+## CI services
+
+For every pull request and git push to the master repository, a number of
+build and testing jobs are run on a set of different CI services. The exact
+services vary over time. GitHub Actions and AppVeyor are the primary ones
+these days.
+
+## OSS-Fuzz
+
+Google runs the [OSS-Fuzz](https://google.github.io/oss-fuzz/) project which
+also runs fuzzing on curl code, non-stop, in their infrastructure and they
+send us emails in the rare instances they actually find something.
+
+## Coverity
+
+We regularly run our code through the [Coverity static code
+analyzer](https://scan.coverity.com/) thanks to them offering this service to
+us for free.
+
+## CodeSonar
+
+[CodeSonar](https://codesecure.com/our-products/codesonar/) analyzes the curl
+source code daily and emails Daniel Stenberg whenever it finds suspected
+problems in the source code.
+
+## Domain names
+
+The project runs services and website using a few different curl related
+domain names, including `curl.se` and `curl.dev`. Daniel Stenberg owns these
+domain names.
+
+Until a few years ago, the curl website was present at `curl.haxx.se`. The
+`haxx.se` domain is owned by Haxx AB, administrated by Daniel Stenberg. The
+curl.haxx.se name is meant to keep working and be redirecting to curl.se for
+the foreseeable future.
+
+## Websites
+
+The main curl website at `curl.se` is maintained by curl maintainers and the
```

Comment:
is it worth adding a section on tls certs for things like websites, etc ?

Reply:
Good idea, I'll add cert details!